tests/jupyter.rst

```diff
@@ -3,12 +3,14 @@ Jupyter Directive
 
 This is a set of tests related to the Jupyter directive
 
+
 The following jupyter directive with cell-break option should
 split this text and the text that follows into different IN
 blocks in the notebook
 
 .. jupyter::
    :cell-break:
+   :slide-type: subslide
```

Comment:
@FelipeMaldonado I don't see the matching `ipynb` solution notebook update. Is this not running on this notebook? If so we should delete it and keep it separate to the ``cell-break`` example.

Reply:
Oh, I forgot that it had that line, as you see, as I didn't include the ``metadata`` option at the beginning, the extension treats this .rst as a normal notebook. I can delete it